self-core-impl/src/test/java/com/selfxdsd/core/mock/MockJsonResources.java

```diff
@@ -0,0 +1,215 @@
+package com.selfxdsd.core.mock;
+
+import com.selfxdsd.core.JsonResources;
+import com.selfxdsd.core.Resource;
+
+import javax.json.Json;
+import javax.json.JsonArray;
+import javax.json.JsonObject;
+import javax.json.JsonValue;
+import java.net.URI;
+import java.util.function.Function;
+
+/**
+ * A mock implementation of {@link JsonResources} used to unit test
+ * {@link com.selfxdsd.api.Provider} requests.
+ * @author criske
+ * @version $Id$
+ * @since 0.0.8
+ */
+public final class MockJsonResources implements JsonResources {
```

Comment:
@criske This class looks interesting, but it's not clear to me how it will be used. Can you update the JavaDoc here and write an example of usage? You can write code in JavaDocs between ``<pre>`` tags and indentation will be preserved.